README.md

```diff
@@ -0,0 +1,156 @@
+
+Where can I ask general questions about Triton and Triton backends?
+Be sure to read all the information below as well as the [general
+Triton documentation](https://github.com/triton-inference-server/server#triton-inference-server)
+available in the main [server](https://github.com/triton-inference-server/server)
+repo. If you don't find your answer there you can ask questions on the
+main Triton [issues page](https://github.com/triton-inference-server/server/issues).
+
+## Building the vLLM Backend
+
+There are several ways to take advantage of the vLLM backend.
+
+### Option 1. Run the Docker Container.
+
+Pull the container with vLLM backend from [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver) registry. This container has everything you need to run your vLLM model.
+
+### Option 2. Build a custom container from source
+You can follow steps described in the
+[Building With Docker](https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/build.md#building-with-docker)
+guide and use the
+[build.py](https://github.com/triton-inference-server/server/blob/main/build.py)
+script.
+
+A sample command to build a Triton Server container with all options enabled is shown below. Feel free to customize flags according to your needs.
+
+```
+./build.py -v  --enable-logging --enable-stats --enable-tracing
+                --enable-metrics --enable-gpu-metrics --enable-cpu-metrics
+                --enable-gpu
+                --filesystem=gcs --filesystem=s3 --filesystem=azure_storage
+                --endpoint=http --endpoint=grpc --endpoint=sagemaker --endpoint=vertex-ai
+                --backend=python:r23.10
+                --backend=vllm:r23.10
+```
+
+### Option 3. Add the vLLM Backend to the Triton Container
+
+You can install the vLLM backend directly into the NGC Triton container.
+In this case, please install vLLM first. You can do so by running
+`pip install vllm==<vLLM_version>`. Then, set up the vLLM backend in the
+container with the following commands:
+
+```
+mkdir -p /opt/tritonserver/backends/vllm
+wget -P /opt/tritonserver/backends/vllm https://raw.githubusercontent.com/triton-inference-server/vllm_backend/main/src/model.py
+```
+
+## Using the vLLM Backend
+
+You can see an example
+[model_repository](samples/model_repository)
+in the [samples](samples) folder.
+You can use this as is and change the model by changing the `model` value in `model.json`.
+`model.json` represents a key-value dictionary that is fed to vLLM's AsyncLLMEngine when initializing the model.
+You can see supported arguments in vLLM's
+[arg_utils.py](https://github.com/vllm-project/vllm/blob/main/vllm/engine/arg_utils.py).
+Specifically,
+[here](https://github.com/vllm-project/vllm/blob/ee8217e5bee5860469204ee57077a91138c9af02/vllm/engine/arg_utils.py#L11)
+and
+[here](https://github.com/vllm-project/vllm/blob/ee8217e5bee5860469204ee57077a91138c9af02/vllm/engine/arg_utils.py#L201).
+
```

Comment:
@oandreeva-nv  Let's also point some important options like we do in the tutorial. 

```
For multi-GPU support, EngineArgs like tensor_parallel_size can be specified in [vllm_engine_args.json](https://github.com/triton-inference-server/tutorials/blob/main/Quick_Deploy/vLLM/model_repository/vllm/vllm_engine_args.json).

Note: vLLM greedily consume up to 90% of the GPU's memory under default settings. This tutorial updates this behavior by setting gpu_memory_utilization to 50%. You can tweak this behavior using fields like gpu_memory_utilization and other settings in [vllm_engine_args.json](https://github.com/triton-inference-server/tutorials/blob/main/Quick_Deploy/vLLM/model_repository/vllm/vllm_engine_args.json).
```
replace the vllm_engine_args.json with model.json.

Reply:
Thank you, added.